tests/providers/google/cloud/utils/gcp_authenticator.py

```diff
@@ -47,6 +47,7 @@
 GCP_SPANNER_KEY = 'gcp_spanner.json'
 GCP_TASKS_KEY = 'gcp_tasks.json'
 GMP_KEY = 'gmp.json'
+G_FIREBASE_KEY = 'g_firebase.json'
```

Comment:
```suggestion
G_FIREBASE_KEY = 'gcp_firebase.json'
```
Let's keep consistency in case of GCP :)

Reply:
It's not GCP, but Firebase service.  This database is available in the Pantheon, but is most often used by Firebase and was created as part of Firebase. It also has a common name with **Fire**base.